tests/distributions/dist_fixture.py

```diff
@@ -84,10 +84,10 @@ def get_scipy_logpdf(self, idx):
         dist_params = self._convert_logits_to_ps(dist_params)
         args, kwargs = self.scipy_arg_fn(**dist_params)
         if self.is_discrete:
-            log_pdf = self.scipy_dist.logpmf(self.get_test_data(idx, wrap_tensor=False), *args, **kwargs)
+            log_prob_sum = self.scipy_dist.logpmf(self.get_test_data(idx, wrap_tensor=False), *args, **kwargs)
         else:
-            log_pdf = self.scipy_dist.logpdf(self.get_test_data(idx, wrap_tensor=False), *args, **kwargs)
-        return np.sum(log_pdf)
+            log_prob_sum = self.scipy_dist.logpdf(self.get_test_data(idx, wrap_tensor=False), *args, **kwargs)
+        return np.sum(log_prob_sum)
```

Comment:
The `np.sum()` here suggests that the variable name shold be `log_prob` rather than `log_prob_sum`